Formula/c/cppcheck.rb

```diff
@@ -6,6 +6,11 @@ class Cppcheck < Formula
   license "GPL-3.0-or-later"
   head "https://github.com/danmar/cppcheck.git", branch: "main"
 
+  livecheck do
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
  livecheck do
```

The formula's `stable` URL is a tag archive (not a GitHub release asset), so we should document why `GithubLatest` is used here.